format.md

```diff
@@ -49,51 +49,67 @@ The terminology for array types in this section is based on the
 GeoArrow proposes a packed columnar data format for the fundamental geometry
 types, using packed coordinate and offset arrays to define geometry objects.
 
-The inner level is always a FixedSizeList array storing the interleaved
-coordinate values (`[x, y, x, y, ...]`, physically stored as a single array
-of length `n_dim * n_coords`). For any geometry type except Point, this inner
-level is nested in one or multiple
+The inner level is always an array of points. For any geometry type except
```

Comment:
Total nit: I like having a distinction between "point" and "coordinate", where "point" is a geometry type and "coordinate" is just a minimal storage container. For example, the GeoJSON spec first defines ["position"](https://www.rfc-editor.org/rfc/rfc7946#section-3.1.1) and then separately [defines points in terms of positions](https://www.rfc-editor.org/rfc/rfc7946#section-3.1.2).

(This is also why I [didn't originally understand](https://github.com/geopandas/geo-arrow-spec/issues/20#issuecomment-1125311940) that https://github.com/geopandas/geo-arrow-spec/issues/16 referred to all geometry types, not just points)